### ISSUE TITLE
Modern Fortran in Science and Technology by Lindemann and Dahlblom

### ISSUE THREAD

**Issue by Beliavsky** [fortran-lang/fortran-lang.org#285]
[Modern Fortran in Science and Technology](https://modern-fortran-in-science-and-technology.readthedocs.io/en/latest/) could be added to the Online Courses section. I skimmed it, and it seems pretty thorough.

**Comment by certik** [comment]
Yes, we should. Although the first place I randomly clicked: https://modern-fortran-in-science-and-technology.readthedocs.io/en/latest/chapter_fortran.html#arrays-and-matrices and it has a mistake, they assign `5.0` to a double precision number, which will only assign the first 7 digits or so instead of 15.

**Comment by Beliavsky** [comment]
They illustrate this issue with a sample program

```
program constants
    implicit none
    integer, parameter :: dp = selected_real_kind(15,300)
    real(dp) :: pi1, pi2
    pi1 = 3.141592653589793
    pi2 = 3.141592653589793_dp
    write(*,*) 'pi1 = ', pi1
    write(*,*) 'pi2 = ', pi2
    stop
end program constants
```

**Comment by certik** [comment]
Ok, I submitted a PR: https://github.com/jonaslindemann/modern_fortran_book/pull/1

**Comment by jonaslindemann** [comment]
I will look into the issues.

**Comment by jonaslindemann** [comment]
I hope I have fixed it in most places. 

Thanks for the input.

**Comment by certik** [comment]
Thanks @jonaslindemann!

Yes, we should link it.

Btw, currently when I open the page: https://modern-fortran-in-science-and-technology.readthedocs.io/, I am getting "Warning: Potential Security Risk Ahead" in Firefox and Chrome and it does not open for me. It could be an issue on my end.

**Comment by arjenmarkus** [comment]
I just tried with FireFox myself and had no such problem.

Op di 29 jun. 2021 om 15:48 schreef Ondřej Čertík ***@***.***

**Comment by jonaslindemann** [comment]
I don't have any problems myself. It is built and hosted automatically by readthedocs using Sphinx. Thanks for linking it by the way!

Jonas

**Comment by arjenmarkus** [comment]
No problem with MS Edge either, so it seems indeed to be on your side (or
it is a temporary thing)

Op di 29 jun. 2021 om 15:50 schreef Arjen Markus ***@***.***

**Comment by certik** [comment]
Perfect, I am glad it is only on my end. I'll try again in a few days.
